libr/core/cmd_type.c

```diff
@@ -652,16 +659,21 @@ static int cmd_type(void *data, const char *input) {
 		}
 		break;
 	case 's': { // "ts"
-		char *name = strchr (input, ' ');
+		char *tmp = strdup (input);
+		char *name = r_str_trim (strchr (tmp, ' '));
```

Comment:
this is not how r_str_trim works

Reply:
you dont need to strdup anything. just skip the spaces if any, and this can be done in readonly.

Reply:
char *name = r_str_trim_ro (input+1);